### DISCUSSION TITLE
paidPartial email trigger

### DISCUSSION THREAD

**Discussion by earlingeling** [btcpayserver/btcpayserver#5920] [Ideas / Feature Requests]
Sometimes customers pay insufficient funds and get their invoice marked as New (paidPartial).

The reasoning varies - if they use large crypto-exchanges like binance, coinbase and similar, they often take a rather high fee. Also, they might also take their flat btc fee out of the receivers end of the transaction.

To remedy these situations it would be really helpful to be able to automatically trigger an email rule for paidPartial.
Then the store can give in-depth instructions on how this happens and how they can solve it themselv - and largely remove customers enquiries about paidPartial invoices.

Also, an {Invoice.url} link to their invoice would be helpful so we can link them back to their invoice (especially for stores with modal btcpay overlays at checkout)

**Comment by dennisreimann** [comment]
Valid and good use case. Agree we should support this.

@pavlenex let's turn this into an issue.

**Reply by earlingeling** [reply]
Great! :)
Did this get moved as an issue yet?

I'd do it myself, but not too familiar with github or your repo's rules :)
